src/caselawclient/models/judgments.py

```diff
@@ -23,17 +25,19 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         super(Judgment, self).__init__(self.document_noun, *args, **kwargs)
 
     @cached_property
-    def neutral_citation(self) -> str:
-        return self.body.get_xpath_match_string(
-            "/akn:akomaNtoso/akn:*/akn:meta/akn:proprietary/uk:cite/text()",
-            {
-                "uk": "https://caselaw.nationalarchives.gov.uk/akn",
-                "akn": "http://docs.oasis-open.org/legaldocml/ns/akn/3.0",
-            },
+    def neutral_citation(self) -> NeutralCitationString:
+        return NeutralCitationString(
+            self.body.get_xpath_match_string(
+                "/akn:akomaNtoso/akn:*/akn:meta/akn:proprietary/uk:cite/text()",
+                {
+                    "uk": "https://caselaw.nationalarchives.gov.uk/akn",
+                    "akn": "http://docs.oasis-open.org/legaldocml/ns/akn/3.0",
+                },
+            )
         )
 
     @property
-    def best_human_identifier(self) -> str:
+    def best_human_identifier(self) -> NeutralCitationString:
```

Comment:
This should still be a `str`; the fact it's an NCN right now doesn't mean it won't be something less specific in future.
```suggestion
    def best_human_identifier(self) -> str:
```

Reply:
I think this one is still missed!